plugins/flutter_aepedge/README.md

```diff
@@ -174,23 +174,90 @@ final ExperienceEvent experienceEvent = ExperienceEvent({
 });
 List<EventHandle> result = await Edge.sendEvent(experienceEvent);
 ```
+
+**Example with Datastream ID override**
+```dart
+Map<String, dynamic> xdmData = {"eventType": "SampleEventType"};
+final ExperienceEvent experienceEvent = ExperienceEvent({
+  "xdmData": xdmData,
+  "datastreamIdOverride": "SampleDataStreamId"
+});
+List<EventHandle> result = await Edge.sendEvent(experienceEvent);
+```
+
+**Example with Datastream confg override**
+```dart
+Map<String, dynamic> xdmData = {"eventType": "SampleEventType"};
+Map<String, dynamic> configOverrides = {"config": {
+      "com_adobe_experience_platform": {
+        "datasets": {
+          "event": {
+            "datasetId": "sampleDatasetID"
+          }
+        }
+      }
+    }};
+final ExperienceEvent experienceEvent = ExperienceEvent({
+  "xdmData": xdmData,
+  "datastreamConfigOverride": configOverrides
+});
+List<EventHandle> result = await Edge.sendEvent(experienceEvent);
+```
 ------
 ### Public classes
 #### ExperienceEvent
 Experience Event is the event to be sent to Adobe Experience Platform Edge Network. The XDM data is required for any Experience Event being sent using the Edge extension.
 
-##### Create Experience Event from Dictionary:
+You can invoke Experience Event either by using dictionaries or by utilizing convenience constructors.
 
+**Usage**
 ```dart
-Map<String, dynamic> xdmData = {"eventType": "SampleEventType"};
+//Invoke Experience Event by using dictionaries
+
+// set free form data to the Experience event:
+final ExperienceEvent experienceEvent = ExperienceEvent({
+  "xdmData": xdmData,
+  "data": data
+});
+
+// Set free form data and datastream id override to the current Experience event:
 final ExperienceEvent experienceEvent = ExperienceEvent({
-  "xdmData": xdmData
+  "xdmData": xdmData,
```

Comment:
use same examples in sendEvent and here. If possible we should not duplicate the samples at all.